.github/workflows/kind-e2e.yaml

```diff
@@ -69,20 +69,12 @@ jobs:
       uses: actions/checkout@v2
 
     - name: Install KinD
-      run: |
-        set -x
-
-        # Disable swap otherwise memory enforcement doesn't work
-        # See: https://kubernetes.slack.com/archives/CEKK1KTN2/p1600009955324200
-        sudo swapoff -a
-        sudo rm -f /swapfile
-
-        curl -Lo ./kind https://github.com/kubernetes-sigs/kind/releases/download/${{ matrix.kind-version }}/kind-$(uname)-amd64
-        chmod +x ./kind
-        sudo mv kind /usr/local/bin
-
-    - name: Create KinD Cluster
-      run: ./hack/create-kind-cluster.sh
+      # TODO: replace with chainguard-dev/actions/setup-kind
+      uses: chainguard-dev/actions/setup-kind@ce51233d303aed2394a9976e7f5642fd2158f693 # main
```

Comment:
We could go with a slightly never revision than `ce51233d`, e.g. current main has never kube patch versions.

https://github.com/chainguard-dev/actions/blob/ce51233d303aed2394a9976e7f5642fd2158f693/setup-kind/action.yaml#L104

https://github.com/chainguard-dev/actions/blob/main/setup-kind/action.yaml